site/public/js/drag-and-drop.js

```diff
@@ -1288,8 +1288,12 @@ function handleEditCourseMaterials(csrf_token, hide_from_students, id, sectionsE
     formData.append('release_time',cmTime);
     formData.append('sort_priority',priority);
     formData.append('sections_lock', sections_lock);
-    formData.append('link_url', link_url);
-    formData.append('link_title', link_title);
+    if(link_url != null) {
```

Comment:
Can this be changed to `!==` as well as line 1294

Reply:
Should I change the `!=` on line 1297 to `!==` as well?